### PR TITLE
AppDynamics Agent

### PR DESCRIPTION
**Pull request by nkatre**
- Change download location of AppDynamics agents so that latest agents
    are available for download
  - Change AppDynamics App Name to <pcf_space_name>:<pcf_app_name>
  - Enable AppDynamics download

### CODE REVIEW

**Comment by nebhale** [comment]
@nkatre Please sign the Cloud Foundry CLA so that I can include this contribution.

**Comment by nebhale** [comment]
Also, can you please update your repositories `index.yml` so that it is a better formed YAML document.  This means a `---` at the top and no newlines between entries.  Something more like:

```
---
4.1.3_1: https://packages.appdynamics.com/java/4.1.3.1/AppServerAgent-4.1.3.1.zip
4.1.7_1: https://packages.appdynamics.com/java/4.1.7.1/AppServerAgent-4.1.7.1.zip
4.1.7_2: https://packages.appdynamics.com/java/4.1.7.2/AppServerAgent-4.1.7.2.zip
4.1.8_1: https://packages.appdynamics.com/java/4.1.8.1/AppServerAgent-4.1.8.1.zip
4.1.8_2: https://packages.appdynamics.com/java/4.1.8.2/AppServerAgent-4.1.8.2.zip
...
```

**Comment by nkatre** [comment]
@nebhale I am included in the Corporate CLA but didn't publicize the membership. I have publicized my membership now. This should solve the CLA issue.
Regarding the changes to `index.yml`, the changes should be visible once a new release of AppDynamics agent is out, since the changes to `index.yml` file are tied to our `release` branch. I will notify here once the changes are visible.

**Comment by nebhale** [comment]
Looks good @nkatre.  Thanks.

**Comment by h0nIg** [comment]
@nkatre @nebhale whats the status? will this get merged and will the next release include this?

**Comment by nebhale** [comment]
It will be merged, but not until the `index.yml` is updated.  Per your statement, I'm waiting on notification at which point I will check again.  I cannot guarantee that it will be in the next release, as I don't know when the `index.yml` will be updated.

File: config/app_dynamics_agent.yml [hunk 1]
```diff
@@ -16,8 +16,9 @@
 # Configuration for the AppDynamics framework
 ---
 version: 4.+
-repository_root: ""
-default_application_name:
+repository_root: "http://packages.appdynamics.com/java"
```
Comment (nebhale) [inline config/app_dynamics_agent.yml]: This URL should be an `https://` URL.

**Comment by nkatre** [comment]
@nebhale The download link is modified to be `https`
@h0nIg The `index.yml` file is tied to our `release` branch which should be cut in the next 2-3 weeks.

**Comment by nkatre** [comment]
@nebhale The `index.yml` has been fixed. Can you please review

**Comment by h0nIg** [comment]
@nebhale i would appreciate if this will get included into the next release

**Pull request closed without merging**